crates/forge/tests/integration/gas.rs

```diff
@@ -0,0 +1,117 @@
+use indoc::indoc;
+use test_utils::running_tests::run_test_case;
+use test_utils::{assert_gas, assert_passed, test_case};
+
+#[test]
+fn test_keccak_cost() {
+    let test = test_case!(indoc!(
+        r"
+            #[test]
+            fn test_keccak_builtin() {
+                keccak::keccak_u256s_le_inputs(array![1].span());
+            }
+        "
+    ));
+
+    let result = run_test_case(&test);
+
+    assert_passed!(result);
+    assert_gas!(result, "test_keccak_builtin", 20.48);
```

Comment:
Where did these results come from?

Reply:
https://book.starknet.io/ch03-01-02-fee-mechanism.html#computation

from the table at the bottom

Reply:
Let's maybe add the link in a comment

Reply:
I think we have this table in the code somewhere anyways xD The only library it is defined in is `cairo-lang` (yes, the python one xD), even blockifier uses bindings to get it from there xD